bluemira/equilibria/fem_fixed_boundary/file.py

```diff
@@ -145,9 +146,9 @@ def save_fixed_boundary_to_file(
         nz=nz,
         pressure=pressure,
         pprime=p_prime,
-        psi=psi,
+        psi=2.0 * np.pi * psi,
         psibdry=0,
-        psimag=psi_mag,
+        psimag=2.0 * np.pi * psi_mag,
```

Comment:
Currently for better or worse the COCOS of bluemira is 3 which means that the output is in V.s/rad. Feeds into #318 and our various internal problems with what standard we are using

Reply:
also I should add that IMAS IDS has moved from /is moving to COCOS 1 which is also V.s/rad

Reply:
OK, I was just preserving existing behaviour, but I have removed now.